go/libcmdline/cmdline.go

```diff
@@ -343,6 +347,10 @@ func (p *CommandLine) PopulateApp(addHelp bool, extraFlags []cli.Flag) {
 			Usage: "Specify an alternate API URI path prefix.",
 		},
 		cli.StringFlag{
+			Name:  "app-start-mode",
+			Usage: "specify 'service' to auto-start UI app, or anything else to disable",
```

Comment:
Maybe title case (like above).. and maybe fix to be consistent below.

I would say:
```
Specify 'service' to automatically start the UI app when the service starts up or 'disabled' to disable it
```